runtime/v1/linux/runtime.go

```diff
@@ -330,6 +330,10 @@ func (r *Runtime) loadTasks(ctx context.Context, ns string) ([]*Task, error) {
 			continue
 		}
 		id := path.Name()
+		// skip hidden directories
+		if len(id) > 0 && id[0] == '.' {
+			continue
```

Comment:
Should we try deleting hidden directory?

Reply:
Not sure, but the current logic is if we remove hidden directory,  it can not found bundle, since bundle become `xxx/.id`